### ISSUE TITLE
Flexible time pomodoro breaks

### ISSUE THREAD

**Issue by aguscaba** [johannesjo/super-productivity#2079]
Hello, first of all want to thank you for such a wonderful tool

This is a feature request, not an issue or bug

The concise request is to have flexible timing for pomodoro breaks.

Description: I frequently got motivated and would like to continue with what I am doing for a short period of time or simply don't want to cut the idea, actually whenever you get the break reminder (which stops the working timer) you only can take it or skip (lose the break which Is the point of the pomodoro).

It would be great if if you only get the notification saying that you need to take a break with the "Start" and "Skip" buttons (Kindly look at the below image as an example)(meanwhile the actual work timer continues running) then only when you click the start button (to start the break) the work timer stops and the break timer starts.

![2022-05-18 10_54_51-Super Productivity](https://user-images.githubusercontent.com/67975/169091750-f66a9261-73b7-4e95-99ea-3b494ebb8866.jpg)

**Comment by johannesjo** [comment]
Thanks for the suggestion! Makes sense to me, to add something like this! PRs are welcome!

**Comment by Viniuau** [comment]
This is pretty much adding FlowTime, which I'd love to see it being added. I found that Pomodoro was getting in the way. I do creative work, and many times now I've straight up skipped break to not lose focus on my work.

If I may add to this issue, here's what FlowTime is about https://medium.com/@UrgentPigeon/the-flowtime-technique-cheat-sheet-30168b2e31d9#.5kzhko7ab
You tool is already helping me a lot as-is, and I found that the flexible nature of this technique would fit the customizability of your app pretty well.
